UnityProject/Assets/Scripts/Weapons/Gun.cs

```diff
@@ -652,6 +651,13 @@ private void DequeueAndProcessServerShot()
 			{
 				b.Suicide(shooter, this, damageZone);
 			}
+			else if (ProjectilesFired != 1)
+			{
+				for (int n = 0; n > ProjectilesFired; n++)
+				{
+					b.Shoot(ApplyRecoil(finalDirection), shooter, this, damageZone);
```

Comment:
So the recoil is applied twice, because that passed direction is off set with a recoil.

Reply:
Bullets will be offset a little bit. Better to rotate their direction and make rotation dependent on count of projectiles fired.